src/main/java/ch/njol/skript/conditions/CondIsValid.java

```diff
@@ -7,20 +7,29 @@
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
+import org.skriptlang.skript.util.Validated;
 
 @Name("Is Valid")
-@Description("Checks whether an entity has died or been despawned for some other reason.")
+@Description({
+	"Checks whether something (an entity, a script) is valid.",
+	"An invalid entity may have died or de-spawned for some other reason.",
+	"An invalid script reference may have been reloaded, moved or disabled since."
+})
 @Examples("if event-entity is valid")
-@Since("2.7")
-public class CondIsValid extends PropertyCondition<Entity> {
+@Since("2.7, INSERT VERSION (Scripts)")
+public class CondIsValid extends PropertyCondition<Object> {
```

Comment:
might it be good to support config/nodes? or really, just a type for the Validated interface